lib/link.js

```diff
@@ -1,66 +1,134 @@
-// link with no args: symlink the folder to the global location
-// link with package arg: symlink the global to the local
+'use strict'
+
+const { readdir } = require('fs')
+const { resolve } = require('path')
+
+const Arborist = require('@npmcli/arborist')
+const npa = require('npm-package-arg')
+const rpj = require('read-package-json-fast')
+const semver = require('semver')
 
 const npm = require('./npm.js')
 const usageUtil = require('./utils/usage.js')
 const reifyOutput = require('./utils/reify-output.js')
-const { resolve } = require('path')
-const Arborist = require('@npmcli/arborist')
 
 const completion = (opts, cb) => {
-  const { readdir } = require('fs')
   const dir = npm.globalDir
   readdir(dir, (er, files) => cb(er, files.filter(f => !/^[._-]/.test(f))))
 }
 
 const usage = usageUtil(
   'link',
   'npm link (in package dir)' +
-  '\nnpm link [<@scope>/]<pkg>[@<version>]'
+  '\nnpm link [<@scope>/]<pkg>[@version]'
```

Comment:
Pedantic nit, but I think the `<>` is technically correct here, since the `@` is a literal, and `<version>` is a placeholder.

Reply:
oops, unintended change 😬 will fix